app/cni/pkg/cni/utils_linux.go

```diff
@@ -15,15 +15,15 @@ import (
 // copied from https://github.com/kubernetes/kubernetes/blob/v1.24.3/pkg/util/procfs/procfs_linux.go#L99
 // with small modifications
 // and return their pid(s)
-func pidOf(name string) ([]int, error) {
+func pidOf(name string) ([]int, error) { //nolint:unused
```

Comment:
why do this? this file is now guaranteed to be forgotten if the calls disappear.

Reply:
You are right!